projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -747,6 +748,19 @@ public static function is_enabled_jetpack_recurring_payments() {
 		return $api_available;
 	}
 
+	/**
+	 * Whether to enable the blocks in the editor.
+	 * All Monetize blocks (except Simple Payments) need an active connecting and a user with at least `edit_posts` capability
+	 *
+	 * @return bool
+	 */
+	public static function should_enable_monetize_blocks_in_editor() {
+		$manager                          = new Connection_Manager( 'jetpack' );
+		$jetpack_ready_and_user_connected = $manager->is_connected() && $manager->is_user_connected();
+		$api_available                    = ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || $jetpack_ready_and_user_connected );
```

Comment:
Given that we already use the `Host` class in this file, maybe we can get rid of this defined, and instead rely on `( new Host() )->is_wpcom_simple()`?

Reply:
Yes, sounds good!

Reply:
Pushed in 8fbdb4009dc3c0031c92b34d7bfe1eced43af540